lib/tasks/dev.rake

```diff
@@ -316,6 +316,15 @@ namespace :openfoodnetwork do
       if EnterpriseRole.count < 1
         EnterpriseRole.create!(user: Spree::User.first, enterprise: enterprise2)
       end
+      puts <<WARNING
+
+  Warning: This task is going to be replaced by:
+
+    $ bundle exec rake openfoodnetwork:sample_data:load
+
+  It contains more sample data.
+
+WARNING
```

Comment:
nothing to do in this PR but Rails has tool for this. See https://apidock.com/rails/v3.2.13/ActiveSupport/Deprecation/warn/class